core/src/test/scala/com.snowplowanalytics.stream.loader/BadEventTransformerSpec.scala

```diff
@@ -28,13 +35,11 @@ class BadEventTransformerSpec extends Specification {
     "successfully convert a bad event JSON to an ElasticsearchObject" in {
       val input =
         """{"line":"failed","errors":["Record does not match Thrift SnowplowRawEvent schema"]}"""
-      val result = new BadEventTransformer("snowplow", "bad")
-        .fromClass(input -> JsonRecord(input, None).success)
-      val elasticsearchObject =
-        result._2.getOrElse(throw new RuntimeException("Bad event failed transformation"))
-      elasticsearchObject.getIndex must_== "snowplow"
-      elasticsearchObject.getType must_== "bad"
-      elasticsearchObject.getSource must_== input
+      val result = new BadEventTransformer("snowplow")
+        .fromClass(input -> JsonRecord(parse(input).asInstanceOf[JObject], null).success)
```

Comment:
same with the `asInstanceOf` please don't use them

Reply:
removed.